www/js/configs/app.config.js

```diff
@@ -38,66 +38,95 @@ angular.module("binary").constant("config", {
             name   : "asians",
             value  : "Asians",
             markets: ["volidx", "random"]
+        },
+        {
+            name   : "high_low_ticks",
+            value  : "HIGH/LOW TICKS",
+            markets: ["volidx"]
         }
     ],
     tradeTypes: [
         {
-            name    : "Up",
-            value   : "CALL",
-            digits  : false,
-            category: "UP/DOWN"
+            name         : "Up",
+            value        : "CALL",
+            digits       : false,
+            selected_tick: false,
+            category     : "UP/DOWN"
+        },
+        {
+            name         : "Down",
+            value        : "PUT",
+            digits       : false,
+            selected_tick: false,
+            category     : "UP/DOWN"
+        },
+        {
+            name         : "Asians Up",
+            value        : "ASIANU",
+            digits       : false,
+            selected_tick: false,
+            category     : "Asians"
         },
         {
-            name    : "Down",
-            value   : "PUT",
-            digits  : false,
-            category: "UP/DOWN"
+            name         : "Asians Down",
+            value        : "ASIAND",
+            digits       : false,
+            selected_tick: false,
+            category     : "Asians"
         },
         {
-            name    : "Asians Up",
-            value   : "ASIANU",
-            digits  : false,
-            category: "Asians"
+            name         : "Digit Match",
+            value        : "DIGITMATCH",
+            digits       : true,
+            selected_tick: false,
+            category     : "MATCH/DIFF"
         },
         {
-            name    : "Asians Down",
-            value   : "ASIAND",
-            digits  : false,
-            category: "Asians"
+            name         : "Digit Differs",
+            value        : "DIGITDIFF",
+            digits       : true,
+            selected_tick: false,
+            category     : "MATCH/DIFF"
         },
         {
-            name    : "Digit Match",
-            value   : "DIGITMATCH",
-            digits  : true,
```

Comment:
@nazaninreihani 
It seems that the `digits` property has been removed unwittingly from `Digit Even` and `Digit Differes`